src/app/owner/page.tsx

```diff
@@ -0,0 +1,73 @@
+import Image, { StaticImageData } from 'next/image';
+import about from "@images/about/about1.png";
+
+
+const About = () => {
+  return (
+    <div className="min-h-screen bg-white">
+      <main className="max-w-4xl mx-auto p-4">
+        <nav className="text-sm mb-4 mt-[70px]">
+                  <a href="/" className="font-semibold text-sm leading-6 underline">powrót</a>
+        </nav>
+
+              <h3 className="font-bold text-sm leading-6 mt-[29px]">O nas // Katarzyna Bober </h3>
+        <h1 className="text-center font-dm-sans font-medium text-[44px] leading-[52.8px] tracking-[-0.04em] mt-[40px] ">Katarzyna Bober</h1>
+        <div className="flex justify-center mb-6">
+        </div>
+
+        <div>
```

Comment:
The photo should be 1217px wide as in the figma :)

Reply:
done :)